### PR TITLE
VIP-535: Adds sample validation for ExternalFile.FileUri

### PR DESCRIPTION
**Pull request by lizstarin**

### CODE REVIEW

**Review by alice-dw — APPROVED** [review]
👍

File: test/vip/data_processor/validation/v6/external_file_test.clj [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+             [vip.data-processor.validation.xml :as xml]
+             [vip.data-processor.test-helpers :refer :all]
+             [vip.data-processor.validation.v6.external-file :as v6.external-file]
+             [clojure.core.async :as a]))
+
+(use-fixtures :once setup-postgres)
+
+(deftest ^:postgres validate-no-missing-file-uris-success-test
+  (let [errors-chan (a/chan 100)
+        ctx {:errors-chan errors-chan
```
Comment (dwxander) [inline test/vip/data_processor/validation/v6/external_file_test.clj:14]: meant to ask this earlier - what does "ctx" stand for?

**Review by dwxander — APPROVED** [review]
🗄

File: test/vip/data_processor/validation/v6/external_file_test.clj (hunk 1 above)
Comment (danny-pease) [reply test/vip/data_processor/validation/v6/external_file_test.clj:14]: "context", i believe?

Comment (lizstarin) [reply test/vip/data_processor/validation/v6/external_file_test.clj:14]: That's right—it's basically a big data structure that gets modified as it passes through the pipeline. (Functional programming!)

**Review by danny-pease — APPROVED** [review]
🌸

**Pull request merged**